README.md

```diff
@@ -218,19 +218,45 @@ When running OpenShift in a single VM, it is recommended to reuse the Docker dae
 for pure Docker use-cases as well.
 By using the same docker daemon as Minishift, you can speed up your local experiments.
 
-To be able to work with the docker daemon on your Mac or GNU/Linux host use the
-[`docker-env command`](https://minishift.io/docs/minishift_docker-env.html) in your shell:
+1. Make sure that you have `docker` and `docker-client` installed on your machine.
+
+1. Enable and start the `docker` and `docker-containerd` services.
+
+```
+$ systemctl start docker && systemctl enable docker
```

Comment:
This does not make sense. You are effectively starting the Docker daemon here, right? This is exactly what we not want. You are describing how to install and run the Docker daemon on Fedora, but the whole point is that you want to reuse the Docker daemon of the Minishift VM.